jenkins/create_rivers_basemap

```diff
@@ -0,0 +1,76 @@
+pipeline {
+  agent {
+        node {
+            label 'node:slave'
+        }
+    }
+  parameters {
+        gitParameter name: 'BRANCH_TAG',
+                     type: 'PT_BRANCH_TAG',
+                     defaultValue: 'master'
+        choice(choices: ['test', 'qa', 'prod'], description: 'Tier to deploy tiles to', name: 'TIER')
+  }
+  stages {
+    stage('Clean Workspace') {
+	    steps{
+		    cleanWs()
+	    }
+	  }
+    stage('Checkout repo and pull from S3') {
+      steps {
+        sh 'wget -O DOIRootCA2.cer http://sslhelp.doi.net/docs/DOIRootCA2.cer'
+         checkout([$class: 'GitSCM',
+                          branches: [[name: "${params.BRANCH_TAG}"]],
+                          doGenerateSubmoduleConfigurations: false,
+                          extensions: [],
+                          gitTool: 'Default',
+                          submoduleCfg: [],
+                          userRemoteConfigs: [[url: 'https://github.com/usgs-makerspace/wbeep-processing']]
+                        ])
+        sh 'aws s3 sync s3://prod-owi-resources/resources/Application/wbeep/nhd_plus_stream_order_geojson . --include "*detail.geojson"'
```

Comment:
this should download all 3 of the most_detailed, least_detailed and medium_detailed.geojson files